src/resolve-package.js

```diff
@@ -21,13 +31,16 @@ async function resolvePackage({
     let versions = await getVersions(name);
     let getTagVersion = _getTagVersion(versions, name);
     version = await getTagVersion(range);
+    defaultBlueprintOverride = module.exports.getBlueprintNameOverride(name);
```

Comment:
Maybe this should move to install.js, because there are other consumers of resolvePackage that don't use this (which waits for an npm call)

Reply:
Oh good catch, it's only used in the `install` command